packages/flutter/lib/src/services/binding.dart

```diff
@@ -38,6 +39,7 @@ mixin ServicesBinding on BindingBase, SchedulerBinding {
     SystemChannels.lifecycle.setMessageHandler(_handleLifecycleMessage);
     SystemChannels.platform.setMethodCallHandler(_handlePlatformMessage);
     TextInput.ensureInitialized();
+    Scribble.ensureInitialized();
```

Comment:
With scribble in a separated object you don't need to eagerly initialize `TextInput` I think?